src/listener_pynput.py

```diff
@@ -10,7 +10,36 @@ def __init__(self, model, controller):
         self.listener = keyboard.Listener(on_press=self.on_press, on_release=self.on_release, suppress=False)
         self.listener.start()
         self.getNextCallbacks = []
+        self.key_press_handlers = {
+            self.globalArmKey: self.handle_global_arm_press,
+            # TODO: Put these keys in settings
+            "+": self.handle_next_loadout,
+            "-": self.handle_prev_loadout,
+        }
```

Comment:
Need to be added to settings, with a on/off option

Reply:
fixed